pinc/list_projects.inc

```diff
@@ -37,11 +45,39 @@ function list_projects_tersely($where_condition, $order_clause, $limit_clause)
  *
  * url_base is the URL with the beginning of a query string, eg "foo.php?x=10&amp;" or "foo.php?"
  */
-function list_projects($where_condition, $order_clause, $group_clause, $url_base, $per_page = 20, $offset = 0)
+function list_projects(string $metal, string $order_clause, string $url_base, int $per_page = 20, int $offset = 0)
```

Comment:
I know `$per_page = 20` is overridden in the function call, but if you want the default to be 50 instead of 20, should it be changed here?

Reply:
It actually doesn't matter since the only thing calling it is setting it to a number that isn't 20.